00FAQ

```diff
@@ -2203,8 +2203,15 @@ ______________________________________________________________________
 	current working directory.
 
 	If you want to list instances of open files inside the
-	directory, you need to specify the individual path names
-	of those files, or use the lsof +D and +d options.
+	directory, you need to either specify the individual path names
+	of those files, their common mount point, or use the lsof +D 
+	and +d options.
+	
+	For example, if you wanted to check the files open in your /lsof 
+	folder then the following might be the quickest way to get
+	the answer you are looking for:
+
+	lsof / | grep /lsof | grep -v DIR
```

Comment:
Why not `lsof /lsof`?

Reply:
This section of the FAQ is 

```
3.21.4	Why won't lsof find all the open files in a directory?
```

`lsof /lsof` will give different results from `lsof / | grep /lsof` unless `/lsof` is a mountpoint. 

You would need something like `lsof +D /lsof` to be without `grep` or maybe there is another feature of lsof that I am unaware of which would be similar

Reply:
I see.